tests/functional/test_models.py

```diff
@@ -5,13 +5,26 @@
 except ImportError:
     import unittest
 import os
+import sys
+
+# Add project folder to PATH for avoiding troubles in Django < 1.8
+# It can't find `gallery` app
+sys.path.append('test_project')
 os.environ['DJANGO_SETTINGS_MODULE'] = 'test_project.settings'
 
+# Load models for avoiding 'AppRegistryNotReady: Models aren't loaded yet.'
+# in Django 1.7.10
+import django
+if hasattr(django, 'setup'):
+    django.setup()
```

Comment:
All of this changes look unrelated to the new test. Why all this things happened now?